app/_data/app.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   productName: 'Becoming a teacher design history',
-  description: 'A history of the designs for the Find, Apply, Publish, Manage, Register and Support services',
+  description: 'A history of the designs for the Find, Apply, Publish, Manage, Register, Manage school placements, Claim funding for mentor training and Support services',
```

Comment:
We should use an abbreviated version rather than the whole service name for 'Manage school placements' and 'Claim funding for mentor training.' This will follow the style used for the other services. For example:

```suggestion
  description: 'A history of the designs for the Find, Apply, Publish, Manage, Register, Placements, Claim and Support services',
```

Reply:
How about `A history of the designs for the Becoming a Teacher service line`?

Reply:
I think that tagline simply repeats the site's title, 'Becoming a teacher design history'.

It's worth expanding the title and scope from just ‘design’ to include research, product and development, etc.

Reply:
Something like this might work:

> Capturing the history of product, design, and technology decisions shaping trainee teacher recruitment—from finding courses and applying, to managing applications, placements, funding mentors, and registering trainee teachers.

'Capturing' could be replaced with 'Documenting' or something similar.

Reply:
A history of the product, design, technology and policy decisions shaping the services which enable and support trainee teacher recruitment.

Reply:
Let's go with @olioldfield's suggestion.